src/pysdmx/io/xml/sdmx_two_one/__init__.py

```diff
@@ -0,0 +1 @@
+"""SDMX 2.1 XML reader and writer."""
```

Comment:
This is more a matter of taste, but I tend to prefer shorter package names, whenever possible. An alternative name could be `sdmx21`. This is more a matter of taste, obviously, so feel free to ignore 😊

Reply:
Changed to sdmx21